js/header_dialog.js

```diff
@@ -880,9 +945,10 @@ function HeaderDialog(dialog, onSave) {
             setParameter('motor_poles'                            ,sysConfig.motor_poles, 0);
 
                 /* Booleans */
-        setCheckbox('gyro_cal_on_first_arm'                ,sysConfig.gyro_cal_on_first_arm);
-        setCheckbox('vbat_pid_compensation'                ,sysConfig.vbat_pid_compensation);
-        setCheckbox('rc_smoothing'                                ,sysConfig.rc_smoothing);
+        setCheckbox('gyro_cal_on_first_arm'     ,               sysConfig.gyro_cal_on_first_arm);
+        setCheckbox('vbat_pid_compensation'     ,               sysConfig.vbat_pid_compensation);
```

Comment:
Is this still used?

Reply:
Neither vbat_pid_compensation or rc_smoothing are in the log file. There are a lot of parameters that need to be removed. I will review and compare the full list in a later PR.